src/Interpreters/InterpreterKillQueryQuery.cpp

```diff
@@ -462,6 +533,9 @@ AccessRightsElements InterpreterKillQueryQuery::getRequiredAccessForDDLOnCluster
                 | AccessType::ALTER_MATERIALIZE_COLUMN
                 | AccessType::ALTER_MATERIALIZE_TTL
             );
+    /// todo arthur think about this
```

Comment:
Looks ok, rest of the KILL operations require `ALTER_` access right, or what is it you need to think about?